### PR TITLE
Setup/knex

### LINKED ISSUE #1: Create Database Tables
- [ ] Table 1 - Pot holes
- [ ] Table 2 - Pictures

### LINKED ISSUE #2: Table 1 - Potholes
Columns 
  - [ ] id, increments by one per pot hole
  - [ ] n-s  coordinates
  - [ ] e-w   coordinates  
  - [ ] description

### LINKED ISSUE #3: Table 2 - Pictures (urls)
Columns 
- [ ] id, increments by one per picture
- [ ] url, (string)
- [ ] pothole_id - the id of the pot hole associated. from Table 1

### LINKED ISSUE #4: Create seeds to populate tables
- [ ] Three entries to Table 1
- [ ] Some pics for Table 2

### PR DESCRIPTION
**Pull request by Romeslayer**
#### What's this PR do?

This pull request establishes our database with two tables and some seeded data.

#### How should this be manually tested?

You can test this by first ensuring that postgresql is running and you have a empty database labeled 'potspot'.
Then, after checking you would need to run ```npm i``` 
in the console run ```knex migrate: latest``` and ```knex seed:run``` 

#### Any background context you want to provide?

N/A

#### What are the relevant tickets?

Closes #1 
Closes #2 
Closes #3 
Closes #4 

#### Questions:
N/A

### CODE REVIEW

**Comment by RonLHead** [comment]
Approved.

**Review by J-Dunny — APPROVED** [review]
Looks great! Now to build the server with Express

**Review by zliibbe — APPROVED** [review]
This looks great. Good work establishing the backend database and ensuring our tables appropriately reflect the data we're hoping to share. We essentially coded this together and can all confirm that this PR was working locally prior to being pushed up. I think we're in good standing for working toward MVP. Great work y'all!

**Review by RonLHead — APPROVED** [review]

**Comment by zliibbe** [comment]
Looks good to go. On to Express!

File: database/migrations/20220411153629_initial.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function(knex) {
+  return knex.schema
+    .createTable('potholes', function (table) {
+      table.increments('id').primary();
+      table.string('latitude');
+      table.string('longitude');
+      table.string('description');
```
Comment (robbiejaeger) [inline database/migrations/20220411153629_initial.js:11]: For a `string` column type, I believe the default limit to the number of characters is 255. So if you anticipate needing more characters for this column, consider using a `text` type.

**Pull request merged**